### PR TITLE
Feature 11

### PR DESCRIPTION
**Pull request by Namrata-J**
#13 here in this PR, I have updated the styling and have done some work on improving the contrast ratio.

Please review this PR, thank you!!

### CODE REVIEW

**Review by dev-enforced — APPROVED** [review]

**Review by MaruthiPothuganti — APPROVED** [review]
Refactored well. Code looks Good. PR can be merged.

File: components/alert/no-js-alerts/no-js-alert.html [hunk 1]
```diff
@@ -20,8 +20,11 @@
         <header class="et_landing-page-header main-page-header">
             <div class="et_landing-page-header-inner-container">
                 <!-- header-inner-container child1 starts -->
-                <div class="et_inner-container-child1"><img src="/Images/Expeditech UI-logos.jpeg" alt="Expeditech UI"
-                        width="130" class="et_logo" /></div>
+                <div class="et_inner-container-child1">
+                    <a href="/">
+                        <img src="/Images/Expeditech-UI-logos.png" alt="Expeditech UI" width="130" class="et_logo" />
+                    </a>
```
Comment (mehulsatardekar) [inline components/alert/no-js-alerts/no-js-alert.html:26]: Add rel="noopener" or rel="noreferrer" to any external links to improve performance and prevent security vulnerabilities.

chech this link : https://stackoverflow.com/questions/50709625/link-with-target-blank-and-rel-noopener-noreferrer-still-vulnerable

**Review by mehulsatardekar — APPROVED** [review]
Rest things look good, and good work on adding variations of components.

File: components/alert/no-js-alerts/no-js-alert.html (hunk 1 above)
Comment (Namrata-J) [reply components/alert/no-js-alerts/no-js-alert.html:26]: didn't know about this, thankyou Mehul for pointing and suggesting it out!!

**Pull request merged**